### PR TITLE
CSC-268 update process_request to handle unsupported timezones

### PR DESCRIPTION
**Pull request by rustanacexd**
Use the same action here https://github.com/rustanacexd/django-easy-timezones/blob/eec83e1afd59318a406dd639f34e608699907025/easy_timezones/middleware.py#L160 if the timezone is unsupported.

It is being pulled from Connect  through - https://github.com/collabspot/collabspot-cadence/blob/9b7e1700ef5b21ab2be8929d55629967e0145f00/backend/main/requirements.txt#L17

### CODE REVIEW

File: easy_timezones/middleware.py [hunk 1]
```diff
@@ -147,7 +148,11 @@ def process_request(self, request):
                         tz = lookup_tz_v2(ip)
 
         if tz:
-            timezone.activate(tz)
+            try:
+                timezone.activate(tz)
+            except pytz.exceptions.UnknownTimeZoneError:
```
Comment (amatsukov-sugarcrm) [inline easy_timezones/middleware.py:153]: Why don't you want to check if timezone exists before applying it?
https://pypi.org/project/pytz/
`if timezone_str in pytz.all_timezones:`

Comment (rustanacexd) [reply easy_timezones/middleware.py:153]: @amatsukov-sugarcrm  I already tested this way. Apparently `pytz.all_timezones` has `Asia/Famagusta` but Django  doesn't.

Comment (rustanacexd) [reply easy_timezones/middleware.py:153]: Also python uses `EAFP` convention. https://docs.quantifiedcode.com/python-anti-patterns/readability/asking_for_permission_instead_of_forgiveness_when_working_with_files.html

Comment (rustanacexd) [reply easy_timezones/middleware.py:153]: So I was thinking to do the same action as the timezone does't exist at all if Django doesn't support it.

**Review by amatsukov-sugarcrm — APPROVED** [review]

**Pull request merged**